src/app/contextBroker/contextBroker.cpp

```diff
@@ -265,6 +265,8 @@ bool            statTiming;
 bool            statNotifQueue;
 int             lsPeriod;
 bool            relogAlarms;
+bool            strictIdv1;
```

Comment:
Looking to the name of this variable, I think I prefer -strictNgsiv1Ids instead of -ngsiv1CheckIdFields :)

What do you think?